internal/render/render.go

```diff
@@ -46,6 +50,33 @@ func LoadCustomTemplates(dir string) error {
 	return nil
 }
 
+// LoadCustomTemplatesName allows to load in custom templates of a specified name from the templates directory.
+func LoadCustomTemplatesName(name string) error {
+	f, err := templates.Dir(false, "/").Open(name)
+	if err != nil {
+		return fmt.Errorf("templates.Open: %v", err)
+	}
+
+	files, err := f.Readdir(nFile)
+	if err != nil {
+		return fmt.Errorf("f.Readdir: %v", err)
+	}
+
+	for _, f := range files {
+		text, err := templates.FSString(false, path.Join("/", "testify", f.Name()))
```

Comment:
`"testify"` should be `name` here, right?

Reply:
That's right, thank you :pray: